### PR TITLE
Added token support to 'maintenance mode message' field.

### PR DESCRIPTION
**Pull request by ghost**
Fixes https://github.com/backdrop/backdrop-issues/issues/3058.

### CODE REVIEW

File: core/includes/ajax.inc [hunk 1]
```diff
@@ -590,7 +590,7 @@ function ajax_prepare_response($page_callback_result) {
         break;
 
       case MENU_SITE_OFFLINE:
-        $commands[] = ajax_command_alert(filter_xss_admin(t(config_get('system.core', 'maintenance_mode_message'), array('@site' => config_get('system.core', 'site_name')))));
+        $commands[] = ajax_command_alert(filter_xss_admin(token_replace(config_get('system.core', 'maintenance_mode_message'))));
```
Comment (quicksketch) [inline core/includes/ajax.inc]: Not sure about the removal of t() here. I think it should be left in place inside the token_replace. `token_replace(t($message))`.

**Review by quicksketch — CHANGES_REQUESTED** [review]
In principle this seems good. Code-wise, we need to maintain compatibility with the old messages that used `@site` in them, so a simple update hook that str_replace()'s `@site` with `[site:name]` is in order.

**Comment by ghost** [comment]
Uh oh... I think I stuffed this up... Help!

**Comment by ghost** [comment]
Messed this up...

**Pull request closed without merging**